pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -67,8 +70,8 @@ func NewCertRotationController(
 		certrotation.SigningRotation{
 			Namespace:     operatorclient.OperatorNamespace,
 			Name:          "aggregator-client-signer",
-			Validity:      8 * time.Hour, // to be 10 days
-			Refresh:       4 * time.Hour, // to be 4 days
+			Validity:      30 * rotationDay,
```

Comment:
Suggested to say `ValidityOneMonth`

Reply:
I disagree. the day is the unit we might want to change for testing.

Reply:
in that case it might be handier to use a function that returns a consistent set of values

```
Validity : SigningRotationValidity(),
Refresh: SigningRotationRefresh(),
```

or similar, and have those functions handle the internal logic of time unit. This way you have to modify the values in a single location.